context/QuestsProvider.tsx

```diff
@@ -97,6 +100,20 @@ export const QuestsContextProvider = ({
       });
   }, []);
 
+  useMemo(() => {
+    if (!address) return;
+    fetch(
```

Comment:
I think we have a functions in `services` folder which handles api calls . You can implement a similar function to for this